api_examples/sequential_learning_api_tutorial.ipynb

```diff
@@ -0,0 +1,692 @@
+    "    create_new_version=True,\n",
+    "#   given_dataset_id=1234\n",
+    ")\n",
+    "print(f\"Dataset created: {dataset_id}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Make a data view on Citrination, return/print the ID\n",
+    "# view_id = 1234 # If view exists and you're re-running this notebook\n",
+    "\n",
+    "view_id = build_view_and_get_id(client, dataset_id, \"toy_view\", view_desc=\"toy test view\",\n",
+    "                      input_keys=[\"Property x1\", \"Property x2\"], output_keys=[\"Property y\"])\n",
+    "\n",
+    "print(f\"Data view created: {view_id}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Running design\n",
+    "\n",
+    "Now we call a function that runs the SL process. In short, we run a loop where we do the following:\n",
+    "\n",
+    "1. Submit a design run (and poll status w/ wait times)\n",
+    "1. Get best candidates and record predicted results\n",
+    "1. Measure candidates (using toy function evaluation) and record measured results\n",
+    "1. Add candidates to dataset\n",
+    "1. Retrain on dataset (and poll status w/ wait times)\n",
+    "1. Repeat until max iterations reached"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "sl_results = {}\n",
+    "\n",
+    "best_sl_pred_vals, best_sl_measured_vals = run_design(\n",
+    "    client=client,\n",
+    "    view_id=str(view_id),\n",
+    "    dataset_id=str(dataset_id),\n",
+    "    num_candidates_per_iter=10,\n",
+    "    design_effort=30,\n",
+    "    wait_time=3,\n",
+    "    num_sl_iterations=10,\n",
+    "    input_properties=[\"Property x1\", \"Property x2\"],\n",
+    "    target=[\"Property y\", \"Min\"],\n",
+    "    print_output=True,\n",
+    "    true_function=toy_func,\n",
+    "    score_type=\"MLI\"\n",
+    ")\n",
+    "\n",
+    "# Reset dataset after each SL run\n",
+    "dataset_id = upload_data_and_get_id(\n",
+    "    client,\n",
+    "    \"toy_initial_dataset\",\n",
+    "    \"toy_initial_dataset.json\",\n",
+    "    create_new_version=True,\n",
+    "    given_dataset_id=dataset_id\n",
+    ")\n",
+    "\n",
+    "sl_results[\"predicted\"] = best_sl_pred_vals\n",
+    "sl_results[\"measured\"] = best_sl_measured_vals"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Examining the results\n",
+    "\n",
+    "Finally, we can plot the results: Here, we are plotting the best measured candidate against the number of SL iterations. In this case, lower = better for the measured results (since we are doing function minimization)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sns.lineplot(\n",
```

Comment:
I think I'm just being silly, but why does my output differ from plot for "best predicted value"? My "best predicted value" also never deviated from 14.43 between two runs, even when my "best initial value" in training set was 4.61. Should we be expecting strict improvement? Are these two metrics in fact not the same? It's 100% likely a misunderstanding on my part of what these metrics are.
![screen shot 2019-02-25 at 08 32 16](https://user-images.githubusercontent.com/20807339/53353747-46997980-38da-11e9-88fb-b9f8d9831835.png)

Reply:
This confused me too for the longest time hahaha. It's because the plot shows the measured ground-truth results, and the output during the SL iterations shows the predicted values. i.e., you can do better/worse than predicted.